Description: Update fvwm3.desktop file to include Keyword and Icon
Author: Jaimos Skriletz <jaimosskriletz@gmail.com>
Forwarded: https://github.com/fvwmorg/fvwm3/pull/917
Last-Update: 2023-10-20
diff --git a/Makefile.am b/Makefile.am
index 18d8f8b7..d5570808 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -49,4 +49,5 @@ uninstall-hook:
 	-rmdir @FVWM_DATADIR@
 
 $(XSESSIONFILE): $(XSESSIONFILE).in
-	${SED} -e 's|[@]bindir@|$(bindir)|g' < "$@.in" > "$@"
+	${SED} -e 's|[@]bindir@|$(bindir)|g' \
+		-e 's|[@]datadir@|$(datadir)|g' < "$@.in" > "$@"
diff --git a/contrib/xsession/fvwm3.desktop.in b/contrib/xsession/fvwm3.desktop.in
index a6733b74..5f09837b 100644
--- a/contrib/xsession/fvwm3.desktop.in
+++ b/contrib/xsession/fvwm3.desktop.in
@@ -1,5 +1,7 @@
 [Desktop Entry]
 Name=FVWM3
-Comment=The fvwm3 window manager
+Keywords=Fvwm3 Window Manager
+Comment=F? Virtual Window Manager
+Icon=@datadir@/fvwm3/default-config/images/fvwm-logo-small.png
 Type=Application
 Exec=@bindir@/fvwm3
